varinfo/generate_umm_var.py

```diff
@@ -36,25 +36,28 @@ def generate_collection_umm_var(collection_concept_id: str,
         * collection_concept_id: Concept ID for collection that variables have
           been generated for.
         * cmr_env: URLs for CMR environments (OPS, UAT, and SIT)
-        * auth_header: Authorization HTTP header, containing a LaunchPad
-          token: 'Authorization: <token>'
+        * launchpad_token: A LaunchPad token with no header prefixes:
+            <Launchpad token>
         * publish: Optional argument determining whether to publish the
           generated UMM-Var records to the indicated CMR instance. Defaults to
           False.
 
         Note - if attempting to publish to CMR, a LaunchPad token must be used.
 
     """
+    # Get an EDL token and its header given a LaunchPad token
+    auth_header_edl_token = get_edl_token_header(launchpad_token, cmr_env)
+
     granule_response = get_granules(collection_concept_id, cmr_env=cmr_env,
-                                    auth_header=auth_header)
+                                    auth_header=auth_header_edl_token)
```

Comment:
I am tempted to recommend leaving this `auth_header` as the original input to `generate_collection_umm_var`, rather than making it explicitly the header with the EDL bearer token. I'm not sure that CMR is planning to move to search requiring LaunchPad, but LaunchPad is still a valid option. It's really only the download that definitely needs an EDL bearer token right now.

Reply:
Sounds good, I got a little tripped up by the comment below that says:
`Note - if attempting to publish to CMR, a LaunchPad token must be used.`
So, I thought I should update this here but I will change it back.